src/Education/XamlGenerator/XamlGeneratorHelper.cs

```diff
@@ -100,17 +111,97 @@ private void WriteExtendedDescriptionIfPresent(IRuleInfo ruleInfo)
 
             private void WriteSubTitle(IRuleInfo ruleInfo)
             {
+                var isNewCct = ruleInfo.CleanCodeAttribute.HasValue;
+
+                if (isNewCct)
+                {
+                    WriteCleanCodeHeader(ruleInfo);
+                }
+
                 writer.WriteStartElement("Paragraph");
-                writer.ApplyStyleToElement(StyleResourceNames.Title_Paragraph);
+                writer.ApplyStyleToElement(StyleResourceNames.Subtitle_Paragraph);
+
+                if (!isNewCct)
+                {
+                    WriteSubTitleElement_IssueType(ruleInfo);
+                    WriteSubTitleElement_Severity(ruleInfo);
+                }
```

Comment:
Minor: it's a bit harder to see the logical structure here.

Would it matter if the legacy severity info and rule key_tags were in separate `<Paragraph>`s?
If not, it could be something like:

```
var isNewCct = ruleInfo.CleanCodeAttribute.HasValue;

if (isNewCct)
{
  WriteCleanCodeHeader(ruleInfo);
}
else
{
  WriteLegacySeverityHeader(ruleInfo);
}

// Write rule key + normal tags
writer.WriteStartElement("Paragraph");
writer.ApplyStyleToElement(StyleResourceNames.Subtitle_Paragraph);
WriteSubTitleElement_RuleKey(ruleInfo);
WriteSubTitleElement_Tags(ruleInfo);
writer.WriteEndElement();
```

Reply:
Checked how it would look with 2 paragraphs for old severity - doesn't look good